include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -381,6 +414,22 @@ struct pairwise_alignment_edit_distance_unbanded
     {
         return trace_matrix_type{*this};
     }
+
+    //!\brief Return the end position of the alignment
+    alignment_coordinate end_coordinate()
```

Comment:
can we infer the begin coordinate from the trace back?

Reply:
yeah I wanted to address this after this pr.